bluesky_kafka/tests/test_kafka.py

```diff
@@ -1,94 +1,202 @@
+from functools import partial
+import logging
 import multiprocessing
 import pprint
 import time
 
-import pytest
+import numpy as np
+import pickle
 
-from confluent_kafka import Consumer, KafkaException
+import msgpack
+import msgpack_numpy as mpn
+import pytest
 
-from bluesky_kafka.kafka import Publisher, RemoteDispatcher
-from bluesky.plans import count
+# this caused a utf-8 decode error
```

Comment:
Can these lines just be removed then?

Reply:
I wanted to remember why this method, which is recommended, is not in use. I improved the comment.